src/plugins/workspace/public/components/workspace_form/workspace_collaborator_table.tsx

```diff
@@ -79,27 +82,36 @@ export const WorkspaceCollaboratorTable = ({
 
   const emptyStateMessage = useMemo(() => {
     return (
-      <EuiEmptyPrompt
-        title={
-          <h3>
-            {i18n.translate('workspace.workspaceDetail.collaborator.emptyState.title', {
-              defaultMessage: 'Your workspace doesn’t have any collaborators.',
-            })}
-          </h3>
-        }
-        titleSize="s"
-        body={i18n.translate('workspace.workspaceDetail.collaborator.emptyState.body', {
-          defaultMessage:
-            'Currently you’re the only user who has access to the workspace as an owner. Share this workspace by adding collaborators.',
-        })}
-        actions={
-          <AddCollaboratorButton
-            displayedTypes={displayedCollaboratorTypes}
-            permissionSettings={permissionSettings}
-            handleSubmitPermissionSettings={handleSubmitPermissionSettings}
-          />
-        }
-      />
+      <>
+        <EuiFlexGroup alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiEmptyPrompt
+              title={
+                <h3>
+                  {i18n.translate('workspace.workspaceDetail.collaborator.emptyState.title', {
+                    defaultMessage: 'Your workspace doesn’t have any collaborators.',
+                  })}
+                </h3>
+              }
+              titleSize="s"
+              body={i18n.translate('workspace.workspaceDetail.collaborator.emptyState.body', {
+                defaultMessage:
+                  'Currently you’re the only user who has access to the workspace as an owner. Share this workspace by adding collaborators.',
+              })}
+              actions={
+                <AddCollaboratorButton
+                  displayedTypes={displayedCollaboratorTypes}
+                  permissionSettings={permissionSettings}
+                  handleSubmitPermissionSettings={handleSubmitPermissionSettings}
+                />
+              }
+            />
```

Comment:
Learn more in documentation link is missing?

![image](https://github.com/user-attachments/assets/e34e75f0-c561-46ce-a16b-343a764b0712)

Reply:
The link is still missing, so we can ignore it for now.